client/src/voxel_rendering/block.cpp

```diff
@@ -0,0 +1,20 @@
+#include "block.h"
```

Comment:
What do we plan to do with this block class? It just seems like an over glorified boolean at this point.